src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,11 +5,21 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bucketSize;
+
     public Bulldozer() {
     }
 
     @Override
     public void doWork() {
         System.out.println("Bulldozer started to work");
     }
+
+    public int getBucketSize() {
+        return bucketSize;
+    }
+
+    public void setBucketSize(int bucketSize) {
+        this.bucketSize = bucketSize;
+    }
```

Comment:
constructor-based field initialization is more preffered over setter-based
```suggestion
    private final int bucketSize;

    public Bulldozer(int bucketSize) {
    this.bucketSize = bucketSize;
    }

    @Override
    public void doWork() {
        System.out.println("Bulldozer started to work");
    }

    public int getBucketSize() {
        return bucketSize;
    }
```